### ISSUE TITLE
yo are you actually toby?

### ISSUE THREAD

**Issue by Cyuubi** [TotallyTobyFox/undertale#1]
tryna make sure.

**Comment by Dudejoe870** [comment]
never thought this'd happen, this is the last thing I'd expect to go open-source, but if it's actually Toby, or perhaps someone decompiling the game, either way, cool!

**Comment by ThatNerdyPikachu** [comment]
```diff
diff --git a/README.md b/README.md
index de3c07f..04afc92 100644
--- a/README.md
+++ b/README.md
@@ -1,2 +1,2 @@
 # undertale
-Undertale src code leaked
+Undertale src code
```

**Comment by Dudejoe870** [comment]
Yeah, someone else in a Discord server saw that all the variable names seemed like it was a Decomp, still cool though.

**Comment by tumGER** [comment]
@ThatNerdyPikachu Not really a good indication if I'm being honest. I gotta agree that this feels sketchy however.

**Comment by simontime** [comment]
@Dudejoe870 that would be me

**Comment by simontime** [comment]
i said `likely very very fake due to bad audio files, poor variable names and other misc factors which make it look like someone used a gms decompiler` but i'd love to be proven wrong

**Comment by Dudejoe870** [comment]
I'm attempting a build in GMS 1 Professional now.

**Comment by TotallyTobyFox** [comment]
i dont know, someone just sent me these files. i have no idea how they got them ..

**Comment by Dudejoe870** [comment]
they used a Decompiler

**Comment by Dudejoe870** [comment]
That's how, if you don't know how they got them, it's because it's decompiled.

**Comment by TotallyTobyFox** [comment]
dont decompilers often have names like oObjWhateverName ?

**Comment by Dudejoe870** [comment]
They could've documented it.

**Comment by TotallyTobyFox** [comment]
i guess. seems like alot of effort. but maybe its still helpful ?

**Comment by Dudejoe870** [comment]
Decompilation is a lot of effort, there are people who have disassembled games executables into machine code and wrote C code out of that, and along the way documenting what the game does, so this isn't far-fetched.

**Comment by TotallyTobyFox** [comment]
i see. . well its probably still helpful for some stuff then ^-^

**Comment by tumGER** [comment]
No it's illegal then - This is copyrighted content

**Comment by Dudejoe870** [comment]
Well, yeah, the game is payed for.

**Comment by Dudejoe870** [comment]
paid*

**Comment by TotallyTobyFox** [comment]
well i dropped the "leaked" part from the readme becasue i didnt know if it was the real code or not xD

**Comment by Dudejoe870** [comment]
Again, this is a illegal repository, so I would suggest taking it down.

**Comment by Cyuubi** [comment]
yeah

**Comment by TotallyTobyFox** [comment]
this is just a throwaway account i dont really care what happens here xD

(also)[https://www.youtube.com/watch?v=pMhfbLRoGEw]

**Comment by Dudejoe870** [comment]
You do realize illegal means that Toby could DMCA you, right?

**Comment by Dudejoe870** [comment]
Meaning you could go to court.

**Comment by ThatNerdyPikachu** [comment]
rip

**Comment by GovanifY** [comment]
no, he is silicaandpina
http://web.archive.org/web/20190209011251/https://github.com/TotallyTobyFox/undertale/issues/2
^deleted issue

**Comment by NovaPhoenix** [comment]
silica wuz here

**Comment by tumGER** [comment]
So it has come to censorship by a freaking 9 year old leaker already

**Comment by DiscoTiger** [comment]
Omega poggers. Whoever leaked this gave it to the wrong person.

**Comment by Lustyn** [comment]
just a bad decomp """leaked""" by some skid named silica, move along

**Comment by DiscoTiger** [comment]
I shouldn't even say leaked yeah.

**Comment by T3CHNOLOG1C** [comment]
This👏 is 👏 why 👏 you 👏 shouldn't 👏 post 👏 copyrighted 👏 content 👏 to 👏 GitHub

**Comment by Jan200101** [comment]
imagine being so daft that you post copyrighted material on github without permission

**Comment by TotallyTobyFox** [comment]
Not Toby xD

**Comment by mike309game** [comment]
This is most likely decompiled, because the actual project would most likely have a LOT of comments, and objects, roomsand sprites would've been put inside groups. UndertaleModLib exists you can pretty much write a decompiler with it, and it probably wasn't difficult since GameMaker 1.4 uses files structured like xmls.

**Comment by SuperMarioDaBom** [comment]
![FBI](https://i.imgflip.com/2tc4mw.jpg)
Seriously though, unless you wanna go to court I suggest this whole repo be destroyed.
